public/lab-files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
```

Comment:
When validating, the only error that came was that you set the document's language to English, but used lorem ipsum instead. Use lang="zxx".

Don't forget to validate your code before committing, it will help you avoid the little errors.